lib/parallel_tests/tasks.rb

```diff
@@ -117,21 +117,22 @@ def parse_args(args)
   desc "Update test databases by dumping and loading --> parallel:prepare[num_cpus]"
   task(:prepare, [:count]) do |_,args|
     ParallelTests::Tasks.check_for_pending_migrations
-    # dump schema/structure then load in parallel
-    unless defined?(ActiveRecord::Base)
+    if defined?(ActiveRecord::Base) && [:ruby, :sql].include?(ActiveRecord::Base.schema_format)
+      # fast: dump once, load in parallel
+      task_name = (ActiveRecord::Base.schema_format == :ruby ? "parallel:load_schema" : "parallel:load_structure")
+      Rake::Task["db:#{task_name}:dump"].invoke
```

Comment:
`task_name` is used here, so this will become: `Rake::Task["db:parallel:load_schema:dump"].invoke`